src/state/connection-mixin.ts

```diff
@@ -34,6 +34,7 @@ export const connectionMixin = <T extends Constructor<HassBaseEl>>(
         states: null as any,
         config: null as any,
         themes: null as any,
+        selectedThemeSettings: null,
```

Comment:
Shouldnt `selectedTheme` be removed if we are going to rename it?

Reply:
`selectedTheme` is not this structure init, therefore cannot be removed here. Or am I missing your point?